apps/sms/views/conversation/test/unit/information_test.js

```diff
@@ -427,7 +426,7 @@ suite('Information view', function() {
       });
     });
 
-    test('Outgoing Message report(MMS w/ subject)', function() {
+    test.skip('Outgoing Message report(MMS w/ subject)', function() {
```

Comment:
OK, i think we need to add readTimestamp in https://github.com/mozilla-b2g/gaia/blob/master/apps/sms/views/shared/test/unit/mock_messages.js#L54
According to https://dxr.mozilla.org/mozilla-central/source/dom/webidl/MozMmsMessage.webidl it's never absent so the mock should be correct.